Orleans.Consensus.Contract/Log/LogEntry.cs

```diff
@@ -8,15 +8,16 @@ namespace Orleans.Consensus.Contract.Log
     [Serializable]
     public struct LogEntry<TOperation>
     {
+
         public LogEntry(LogEntryId entryId, TOperation operation)
         {
             this.Id = entryId;
             this.Operation = operation;
         }
 
-        public TOperation Operation { get; }
+        public TOperation Operation { get; set; }
 
-        public LogEntryId Id { get; }
+        public LogEntryId Id { get; set; }
```

Comment:
Could we make these setters private? I am guessing you added them so protobuf can access them